packages/api/src/__test__/constants.test.ts

```diff
@@ -0,0 +1,8 @@
+import { expect, test } from '@jest/globals'
+import { FA } from  '../../../../apps/expo/src/utils/constants';
+
```

Comment:
This is good, move the file to the correct location so jest recognize the test and send stat to code cov